source/features/wait-for-checks.tsx

```diff
@@ -192,6 +192,6 @@ void features.add(import.meta.url, {
 	exclude: [
 		pageDetect.isDraftPR,
 	],
-	deduplicate: 'has-rgh-inner',
+	deduplicate: false,
```

Comment:
![fuckit-well](https://user-images.githubusercontent.com/1402241/182195708-88f200eb-c412-44d7-94dc-934485fbde6e.gif)

Reply:
IT WORKED OMG

https://user-images.githubusercontent.com/1402241/182198961-24b39e5d-82b6-4479-9b88-f0380657ac53.mov